src/egon/data/cli.py

```diff
@@ -141,6 +141,7 @@
     help=("Specify the port on which airflow runs."),
     show_default=True,
 )
+
```

Comment:
Do you wanted to change something in this file? I'm just wondering why this file changed on your branch.

Reply:
No, I didn't want to change anything there. This added line is a typo, I'm sorry. Should I remove it?